documentation/docs/reference/services/Profile.md

```diff
@@ -232,22 +240,24 @@ Response format:
 GET /profile/filtered/?teamStatus=value&interests=value,value,value&limit=value
```

Comment:
Deprecated and no longer exists (GET /profile/list/ took its place instead). Dunno why we didn't remove it then, but please remove it.

Reply:
Literally not even bound to anything. We need a bot to verify documentation was updated lol.

Resolved by acabcca.